guide/getting-started-with-testcontainers-for-dotnet/index.adoc

```diff
@@ -0,0 +1,155 @@
+---
+title: "Getting started with Testcontainers for .NET"
+date: 2023-02-10T09:39:58+05:30
+draft: false
+description: This guide will help you to get started with Testcontainers for .NET by demonstrating how you can use PostgreSQL for testing.
+repo: https://github.com/testcontainers/tc-guide-getting-started-with-testcontainers-for-dotnet
+languages:
+  - C#
```

Comment:
Should the language be ".NET"?

Reply:
No, .NET is the SDK or runtime. C# is the language, there is also VB.NET or  F#.